app/views/spree/admin/orders/_add_product.html.haml

```diff
@@ -4,8 +4,11 @@
   %fieldset.no-border-bottom
     %legend{:align => "center"}= Spree.t(:add_product)
 
-    .field.twelve.columns.alpha{"data-hook" => "add_product_name"}
-      = label_tag :add_variant_id, Spree.t(:name_or_sku)
-      = hidden_field_tag :add_variant_id, "", :class => "variant_autocomplete fullwidth"
-      
+    - if @order.canceled?
+      = t(".cannot_add_item_to_canceled_order")
```

Comment:
If you change this, you also have to change `en.yml`.

We really need to raise errors in feature specs if a translation can't be found...